tests/vue/multiparser/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -164,6 +164,78 @@ printWidth: 80
 ================================================================================
 `;
 
+exports[`unknown.vue format 1`] = `
+====================================options=====================================
+parsers: ["vue"]
+printWidth: 80
```

Comment:
Can you add tests for the case `--vue-indent-script-and-style` is `true`?

Reply:
`embeddedLanguageFormatting: "off"` `vueIndentScriptAndStyle: true` tested